sormas-backend/src/main/java/de/symeda/sormas/backend/travelentry/services/BaseTravelEntryService.java

```diff
@@ -66,6 +72,23 @@ public Predicate createUserFilter(TravelEntryQueryContext qc, boolean checkJuris
 			filter = CriteriaBuilderHelper
 				.and(cb, filter, CriteriaBuilderHelper.limitedDiseasePredicate(cb, currentUser, qc.getRoot().get(TravelEntry.DISEASE)));
 		}
+		if (isRestrictedToAssignedEntities()) {
```

Comment:
limited user should be handled by the `inJurisdictionOrOwned` so this if block is not needed

Reply:
removed the code